causal_testing/testing/estimators.py

```diff
@@ -152,8 +152,7 @@ def _run_logistic_regression(self, data) -> RegressionResultsWrapper:
                 treatment_and_adjustments_cols = pd.get_dummies(
                     treatment_and_adjustments_cols, columns=[col], drop_first=True
                 )
-        regression = sm.Logit(outcome_col, treatment_and_adjustments_cols)
-        model = regression.fit()
```

Comment:
Is it intentional to remove calling the .fit() function?

Reply:
I didn't remove it. I just made it a one liner since the only use of `regression` was to call `regression.fit()`, so I merged the two lines into one